setup.py

```diff
@@ -78,7 +78,7 @@ def run(self):
 
 setup(
     name='edge-genome',
-    version='1.6.0',
+    version='1.6.1',
```

Comment:
Actually, could you remove this line of change? We have bump version script in master to handle the version update. https://github.com/ginkgobioworks/edge#versioning

Reply:
I'll handle the version bump this is merged in.

Reply:
K, I removed the 1.6.0->1.6.1 change and it is building again.